tools/etc/iproute2/rt_protos.d/frr.conf

```diff
@@ -12,3 +12,4 @@
 195  pbr
 196  static
 197  openfabric
+198  path
```

Comment:
hmm  -no, I think 198 is already in use for "SRTE" - see zebra/rt_netlink.h ?

Reply:
path and srte are the same.
we just have to agree on the exported name

Reply:
yes, sorry - it seems like it'd be good to have the frr output align with the iproute2, if possible